gci/src/main/java/com/danielfireman/gci/Sampler.java

```diff
@@ -0,0 +1,49 @@
+package com.danielfireman.gci;
+
+import java.util.concurrent.atomic.AtomicInteger;
+
+public class Sampler {
```

Comment:
Do we need this class to be public? I would say its visibility could be [default](https://www.safaribooksonline.com/library/view/learning-java-4th/9781449372477/ch06s04.html)

Reply:
Done